qemu/tests/cfg/fio_windows.cfg

```diff
@@ -1,16 +1,29 @@
 - fio_windows:
     type = fio_windows
     only Windows
+    image_size_disk1 = 2G
+    create_image_disk1 = yes
```

Comment:
you can remove this line

Reply:
it's better to remove it after test

Reply:
what's the image name

Reply:
the test log stored in data disk after fio on data disk, so I think no need to remove it

Reply:
thanks for your advises, will update the cfg file later~